cli/src/mainframe.tsx

```diff
@@ -41,26 +77,40 @@ const Router = () => {
   )
 }
 
-const Mainframe = () => {
+export interface MainframeProps {
+  command?: CommandKeys
+  options: Record<OptionKey, string>
+  cli?: boolean
+}
+
+const Mainframe: React.FC<MainframeProps> = ({ options, command, cli }) => {
   return (
-    <NotificationProvider>
-      <PublicClientProvider>
-        <WalletProvider>
-          <WalletClientProvider>
-            <CommandProvider>
-              <Router />
-              <CommandControl />
-              <NotificationBar />
-            </CommandProvider>
-          </WalletClientProvider>
-        </WalletProvider>
-      </PublicClientProvider>
-    </NotificationProvider>
+    <CommandProvider options={options} command={command} cli={cli}>
+      <NotificationProvider>
+        <PublicClientProvider>
+          <WalletProvider>
+            <WalletClientProvider>
+              {cli ? (
```

Comment:
Could just hide and disable CommandControl inside its code, and still display notifications in the bottom (maybe add a line that says "Logs:", so we don't need to have two branches here

Reply:
Okay